src/backend/SportLink.Backend/SportLink.API/Services/Organization/OrganizationService.cs

```diff
@@ -0,0 +1,113 @@
+            var owner = await _context.Users.FindAsync(int.Parse(userId!));
+            if (userId is not null && owner is not null)
+            {
+                var organization = new Data.Entities.Organization
+                {
+                    Name = organizationDto.Name,
+                    Description = organizationDto.Description,
+                    ContactEmail = organizationDto.ContactEmail,
+                    ContactPhoneNumber = organizationDto.ContactPhoneNumber,
+                    Location = organizationDto.Location,
+                    OwnerId = int.Parse(userId),
+                    VerificationStatus = VerificationStatusEnum.Unverified,
+                    Owner = owner
+                };
+                await _context.Organizations.AddAsync(organization);
+                await _context.SaveChangesAsync();
+
+                var orgDto = _mapper.Map<OrganizationDto>(organization);
+
+                return new OkObjectResult(orgDto);
+            }
+            else
+            {
+                return new BadRequestResult();
+            }
+        }
+
+        public async Task<List<OrganizationDto>> GetOrganizations(bool isVerified)
+        {
+            if (isVerified)
+            {
+                var organizations = await _context.Organizations.Where(x => x.VerificationStatus == VerificationStatusEnum.Accepted).ToListAsync();
+                return _mapper.Map<List<OrganizationDto>>(organizations);
+            }
+            else
+            {
+                var organizations = await _context.Organizations.Where(x => x.VerificationStatus == VerificationStatusEnum.Unverified).ToListAsync();
+                return _mapper.Map<List<OrganizationDto>>(organizations);
+            }
+        }
+
+        public async Task<OrganizationDto> GetSingleOrganization(int id)
+        {
+            var organization = await _context.Organizations.FindAsync(id);
+            return _mapper.Map<OrganizationDto>(organization);
+        }
+
+        public async Task<bool> VerifyOrganization(int id)
+        {
+            var organization = await _context.Organizations.FindAsync(id);
+            if (organization is null)
+            {
+                return false;
+            }
+            else
+            {
+                organization.VerificationStatus = VerificationStatusEnum.Accepted;
+                await _context.SaveChangesAsync();
+                return true;
+            }
+        }
+
+        public async Task<bool> DeclineOrganization(int id, string reason)
+        {
+            var organization = await _context.Organizations.FindAsync(id);
+            if (organization is null)
+            {
+                return false;
+            }
+            else
+            {
+                organization.VerificationStatus = VerificationStatusEnum.Rejected;
```

Comment:
Needs to check if VerificationStatus is Unverified, then do this. But if it was already verified (declined or rejected), then returns false -> as if "not found"